src/app/messages/en/messages.json

```diff
@@ -59,6 +59,7 @@
   "i18n:footer:social:in:alt": "linkedin",
   "i18n:footer:social:rice-logo:alt": "Rice University logo",
   "i18n:footer:social:tw:alt": "twitter",
+  "i18n:highlighter:display-note:label": "highlight editor",
```

Comment:
Totally nitpicky (sorry lol) but if we can capitalize that it will match the other highlighter messages